### ISSUE TITLE
Allow focus method to be called for household icons

### ISSUE THREAD

**Issue by TerminatorJonny** [WTW-IM/es-components#676]
We need a way to focus input elements (button or radio or checkbox etc.) from es-components. Es-components doesn't currently forward the reference that would allow us to call the focus method.

**Comment by stevematney** [comment]
@TerminatorJonny is there a specific component that isn't supporting ref forwarding that you need to have supported? Many do, but I'm sure there are some that don't.

**Comment by TerminatorJonny** [comment]
@stevematney Yeah the specific component is the household participant Icons. [IconButton](http://localhost:6060/#iconbutton) 

![2022-11-14_11h21_23](https://user-images.githubusercontent.com/47194690/201736844-2c304eb0-11c4-4bcf-aed3-0e5edbd65d36.png)
